bedrock/firefox/views.py

```diff
@@ -712,6 +712,8 @@ def new(request):
                 template = 'firefox/new/berlin/scene1-privat.html'
             elif experience == 'auf-deiner-seite':
                 template = 'firefox/new/berlin/scene1-auf-deiner-seite.html'
+            elif switch('firefox_new_de'):
```

Comment:
Is there a specific reason these templates need switches? Since this is SEO content, it could take days to weeks for search engines to pick up any changes. If we decided to roll this back, we could just as easily remove the templates via a commit / push?

We could then also make these templates locale-specific (name them `scene1.en-GB.html` and `scene1.de.html`), removing the need to do anything fancy in the view here for things to work.